app/api_mocks/party_service_mock.py

```diff
@@ -22,7 +23,7 @@ def get_user_details(self, uuid):
         try:
             return Response(response=json.dumps(self._respondent_ids[uuid]), status=200, mimetype="text/html")
         except KeyError:
-            logger.debug('User ID %s not in mock party service', uuid)
+            logger.debug('User ID not in mock party service', uuid=uuid)
```

Comment:
Would `user_id` be a better key in the message for context? (Or `user_uuid` to be specific)